src/app/components/core/scheduler/list-schedules/list-schedules.component.html

```diff
@@ -25,7 +26,13 @@
             </td>
             <td><small>{{schedule.type}}</small></td>
             <td>
-              <small *ngIf="schedule.repeat.trim() !== '00:00:00'"> {{schedule.repeat.trim()}}</small>
+              <small *ngIf="schedule.repeat.trim() !== '00:00:00' && schedule.type !== 'TIMED'"> {{schedule.repeat.trim()}}</small>
```

Comment:
should not this condition be,

`schedule.type !== 'INTERVAL'" && schedule.repeat.trim() !== '00:00:00' `?